### ISSUE TITLE
The image crops when I use scale

### ISSUE THREAD

**Issue by SamaherBantan** [wonderunit/storyboarder#936]
The image crops when I use scale

 when I draw in the frame then I use the scale and goes out of the frame then when I scale it down it crop 
 I expected to be the same without erasing anything 

My computer’s operating system --> Windows
My computer hardware specs --> Intel core i7 4770 CPU 10 GB RAM

**Comment by archnila** [comment]
I'm also having the same issue. It might be a bug but I'm not sure.

**Comment by DCNielsen** [comment]
I agree. I'd prefer not to destroy artwork when I scale or move the art. I'm guessing there needs to be a canvas/art size cap to keep performance up? But in a "perfect world", Storyboarder knows about the pixels outside my artboard the same way that Photoshop does.
